kinto/integration-test/test_kinto_sync.py

```diff
@@ -0,0 +1,156 @@
+import base64
+import datetime
+import re
+import requests
+import time
+import unittest
+
+from kinto_client import Client
+from kinto_client import replication
+
+class Kinto_Sync(unittest.TestCase):
+    """
+    Tests that given two Kinto servers you can successfully sync data
+    from one to the other
+    """
+
+    def setUp(self):
+        self.master_url = "http://192.168.99.100:8888/v1/"
+        self.read_only_url = "http://192.168.99.100:8889/v1/"
+        self.credentials = ('testuser', 'abc123')
+        self.master = Client(server_url=self.master_url, auth=self.credentials)
+        self.read_only = Client(server_url=self.read_only_url, auth=self.credentials)
+
+        # Try and create our buckets and collections
+        try:
+            self.master.get_bucket('test')
```

Comment:
Not sure if you want to create a constant for `'test'` since I think we use it about ~30 places in this file.